nf_core/sync.py

```diff
@@ -305,18 +280,25 @@ def update_existing_pull_request(self, pr_title, pr_body_text):
             r_json = r.content
             r_pp = r.content
 
-        # PR worked
         if r.status_code == 200:
             log.debug("GitHub API listing existing PRs:\n{}".format(r_pp))
 
             # No open PRs
             if len(r_json) == 0:
-                log.info("No open PRs found between TEMPLATE and {}".format(self.from_branch))
+                log.info("No open PRs found between {} and {}".format(branch, self.from_branch))
                 return False
 
-            # Update existing PR
+            # Close existing PR
+            pr_title = "Important! Template update for nf-core/tools v{} Closed because outdated!".format(
+                nf_core.__version__
+            )
+            pr_body_text = (
+                "A new release of the main template in nf-core/tools has just been released. "
+                "This automated pull-request attempts to apply the relevant updates to this pipeline.\n\n"
+                "This pull-request is outdated and has been closed. A new pull-request has been created instead."
+            )
```

Comment:
🤔  okay I'll see if how to do that

Reply:
The linting code used to post GitHub comments:

https://github.com/nf-core/tools/blob/9b3eec9973160cd8b01e182b8f77ae7de2a73ddc/nf_core/lint.py#L1473-L1525

I removed it because the linting runs on the PR base, so didn't have sufficient authentication when the PR came from someone's fork. But here we have full auth permissions and it's coming from a branch on the same repo so it should work fine I think..

Reply:
Would be nice to resurrect that code and put it to some use again! 😅

Reply:
oh that's nice!! thanks 👌